### ISSUE TITLE
support for where association does not exist

### ISSUE THREAD

**Issue by istrau2** [sequelize/sequelize#5485]
I am using breeze-sequelize and trying to write a statement that returns all records where an association does not exist.

The breeze-sequelize code contains the following line:

```javascript
      if (!_.isEmpty(predSq.include)) {
        throw new Error("Unable to negate an expression that requires a Sequelize 'include'");
      }
```

I am wondering if there is any way in sequelize to negate an association clause, so something like:
Select * from account where account.account_pkey not exists
(select account_pkey from privilege where privilege.user_pkey = 46)

Basically I am trying to write a query for all acounts where a specific user does NOT have a privilege on the account. Is there any way to write such a statement in sequelize?

**Comment by mickhansen** [comment]
Not possible using includes atm

**Comment by swac** [comment]
@mickhansen is this still an issue?
